pastepwn/core/pastepwn.py

```diff
@@ -102,6 +110,12 @@ def start(self):
         self.paste_dispatcher.start()
         self.action_handler.start()
 
+        for onstart_handler in self.onstart_handlers:
+            try:
+                onstart_handler()
+            except Exception as e:
+                self.logger.error("Onstart hander %s failed with error: %s. Pastepwn is still running." % (onstart_handler.__name__, e))
```

Comment:
Small typo "Onstart **hander**". Maybe you could fix it before merging. The rest looks fine :) Thank you!